prime-router/src/main/kotlin/cli/FileUtilities.kt

```diff
@@ -1,150 +1,150 @@
 package gov.cdc.prime.router.cli
 
-import gov.cdc.prime.router.CovidSender
 import gov.cdc.prime.router.FakeReport
 import gov.cdc.prime.router.FileSource
 import gov.cdc.prime.router.Metadata
 import gov.cdc.prime.router.Report
 import gov.cdc.prime.router.SettingsProvider
+import gov.cdc.prime.router.TopicSender
```

Comment:
We should stop using the explicit `CovidSender` if we just want to test topic-based senders.